docs/execution-guide.md

```diff
@@ -29,11 +39,12 @@ If you would like, feel free to confirm that this is the case by inspecting the
 1. To see that the pipeline has executed, search for `Data factories` and go to the Data factories page.![azure-search-data-factories](./images/azure-search-data-factories.png)
 1. Select your data factory, which will be titled `phdi-{environment}-data-factory`.![azure-select-ingestion-pipeline](./images/azure-select-ingestion-pipeline.png)
 1. Launch the Data Factory Studio by clicking the blue button that says `Launch studio`.![azure-data-factory-launch-studio](./images/azure-data-factory-launch-studio.png)
-1. Select your ingestion pipeline, which will be titled `phdi-{environment}-ingestion`.![azure-select-ingestion-pipeline-in-studio](./images/azure-select-ingestion-pipeline-in-studio.png)
+1. Select your ingestion pipeline, which will be titled `phdi-{environment}-ingestion`. If you see multiple piplines with this name, select the one at the top (the most recently opened one).![azure-select-ingestion-pipeline-in-studio](./images/azure-select-ingestion-pipeline-in-studio.png)
```

Comment:
I think we noticed that the pipeline won't be listed if it's your first time opening data factory. So we should add a disclaimer about this and maybe just say to go straight to the Monitor tab if there's no pipelines listed?

Reply:
Sg! I'll add the disclaimer for now and then might ask you a follow up Q about this later on whether it's best to just flip the steps regardless!